exec/java-exec/src/main/java/org/apache/drill/exec/testing/ExecutionControlsInjector.java

```diff
@@ -81,8 +83,20 @@ public void injectPause(final ExecutionControls executionControls, final String
       executionControls.lookupPauseInjection(this, desc);
 
     if (pauseInjection != null) {
-      logger.debug("Pausing at {}", desc);
-      pauseInjection.pause();
+      long pauseDuration = pauseInjection.getMsPause();
+      if ( pauseDuration > 0L) {
+        logger.debug("Pausing at {} for {} sec", desc, TimeUnit.MILLISECONDS.toSeconds(pauseDuration) );
+      } else {
+        logger.debug("Pausing at {}", desc);
+      }
+      try {
+        pauseInjection.pause();
+      } catch (InterruptedException e) {
+        //Unpause if this is a timed pause, because an explicit unpause is not called
+        if (pauseDuration > 0L) {
```

Comment:
maybe to be done in PauseInjection (seems like something internal)

Reply:
Just following convention of 'unpausing' explicitly, rather than implicitly within the `pause()` for a limited-duration pause.

Reply:
Will make the change since the InterruptedException is being handled in the timed pause (based on your comment below).

Reply:
did you forget to remove that part following up on your other change?